tests/e2e/cypress/integration/search.spec.js

```diff
@@ -453,15 +450,15 @@ describe('Search', function() {
 
     cy.get('.RawFilter-submitBtn').click()
 
-    cy.get('.DocumentListItem').should(function($el) {
+    cy.get('.DocumentListItem').should(function ($el) {
       expect($el.first()).to.contain('Maret')
```

Comment:
Shouldn't be like line 385 ?
```suggestion
      expect($el.first()).to.contain('maret')
```

Reply:
No, because in the test at line 329, I changed the fixtures I inject (which, yeah, are not centralized in a proper fixture file, but are repeated in each test, sorry).
Actually, I could have avoided changing these tests. They fail because we ripped out all the Materialize CSS and the jQuery logic, so legacy dropdowns and other "complex" widgets aren't working anymore. Once we migrate the search UI, we'll have to entirely re-write the tests, so my changes are irrelevant at the time being.